### PR TITLE
feat: card page

### PR DESCRIPTION
**Pull request by CallMeSahu**

### CODE REVIEW

File: assets/card/card.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    gap: 1rem;
+}
+
+.container3{
+    /* background-color: var(--secondary-color);  */
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    flex-direction: column;  
+
```
Comment (Avin008) [inline assets/card/card.css:96]: remove blank spaces

**Review by Avin008 — APPROVED** [review]
code is well structured good to merge

File: assets/card/card.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
+.main-content-header{
```
Comment (SuyashTiwariCoder) [inline assets/card/card.css:1]: You can make classes for margin 1rem in your utility classes and add those classes directly here .

File: assets/card/card.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    margin: 1rem;    
+}
+
+.main-content-para{
+   margin: 1rem; 
+}
+
+.card-varient-container{
+    width: 900px; 
+    height: 300px;
```
Comment (SuyashTiwariCoder) [inline assets/card/card.css:11]: use rems instead of pixels . If value is below 1rem then use pixels .

File: assets/card/card.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+.card-title{
+    padding-bottom: 2rem;
+    font-weight: bold;
+    
+} 
+
+.e-commerce{
+    display: flex;
+    flex-direction: column; 
+    padding: 0rem;
```
Comment (SuyashTiwariCoder) [inline assets/card/card.css:68]: If padding is not in use you can remove this attribute .

**Review by SuyashTiwariCoder — APPROVED** [review]

**Pull request merged**